presto-docs/src/main/sphinx/functions/array.rst

```diff
@@ -96,12 +96,10 @@ Array Functions
     Subsets array ``x`` starting from index ``start`` (or starting from the end
     if ``start`` is negative) with a length of ``length``.
 
-.. function:: zip(array1, array2[, array3[, array4]]) -> array<row>
+.. function:: zip(array1, array2[, ...]) -> array<row>
```

Comment:
I'm not sure this is the right change. This is not a var args function. It only work with 2-4 arguments